config/redirects

```diff
@@ -4,6 +4,9 @@ define: manual https://docs.mongodb.com/manual
 define: atlas https://docs.atlas.mongodb.com
 define: devhub https://developer.mongodb.com
 
+# Go
+raw: drivers/go -> ${base}/go/current/
```

Comment:
Question: Any reason why this redirect goes to the current branch? Seems like Java and [Node](https://github.com/mongodb/docs-ecosystem/blob/b78f0518081ab9ffcf176a323c335ec488500974/config/redirects#L57) don't go to a branch?

Reply:
> Question: Any reason why this redirect goes to the current branch? Seems like Java and [Node](https://github.com/mongodb/docs-ecosystem/blob/b78f0518081ab9ffcf176a323c335ec488500974/config/redirects#L57) don't go to a branch?

There's no redirect for Java (java/sync) in this file. We added a different redirect to address how we renamed the drawer that contains all the JVM drivers.

The `drivers/javascript` redirect was likely added when we only had one version of node. We should update that one to current as it currently leads to two redirects (drivers/javascript -> drivers/node -> drivers/node/current).

The Java one is not a redirect to the java/sync driver.